### PR TITLE
implement next content btn

### PR DESCRIPTION
**Pull request by 66eli77**

### CODE REVIEW

**Comment by indirectlylit** [comment]
hey - hope you're feeling a bit better!

Unfortunately (as mentioned in https://github.com/learningequality/kolibri/pull/604) this change would couple the Perseus renderer to the implementation details of the Learn plugin.

we'll probably need to use props (to enable/disabled a 'next content' button) and event-based communication (to communicate to the parent that a button was pressed)

let me know if you'd like to chat about this.

**Comment by 66eli77** [comment]
@indirectlylit that will be great, I need your help to understand how learn page works and implement to next-content feature in learn.

File: exercise_perseus_render/assets/src/vue/perseus.vue [hunk 1]
```diff
@@ -38,6 +37,8 @@
 
 <script>
 
+  const actions = require('kolibri/coreVue/vuex/actions');
```
Comment (indirectlylit) [inline exercise_perseus_render/assets/src/vue/perseus.vue]: I just merged in this PR - https://github.com/learningequality/kolibri/pull/589

so this line will change to:

``` javascript
const actions = require('kolibri.coreVue.vuex.actions');
```

Comment (indirectlylit) [reply exercise_perseus_render/assets/src/vue/perseus.vue]: maybe also better to call this variable `coreActions` ?

**Review by indirectlylit — CHANGES_REQUESTED** [review]
looks good! couple small changes

**Pull request merged**